src/main/java/com/commercetools/sync/sdk2/products/helpers/ProductBatchValidator.java

```diff
@@ -255,7 +254,7 @@ public static Set<String> getReferencedProductKeys(
   private static Set<String> getReferencedKeysWithReferenceTypeId(
       @Nonnull final ProductVariantDraft variantDraft, @Nonnull final String referenceTypeId) {
 
-    final List<AttributeDraft> attributeDrafts = variantDraft.getAttributes();
+    final List<Attribute> attributeDrafts = variantDraft.getAttributes();
```

Comment:
Here is a bug from sdk-v2. It should return `AttributeDraft` actually. Please update sdk-v2 to v10.0.0 in gradle and check it.

Reply:
Even in sdk-v2 version V10.1.0, It returns `Attribute` for me.

Reply:
sorry I messed up with `AssetDraft`

Reply:
As discussed [here](https://commercetools.slack.com/archives/CCJALDJ9F/p1671108559843709?thread_ts=1670409439.148659&cid=CCJALDJ9F) We will be using Attribute only. We no more have AttributeDraft in the latest version.